tests/unit/adapter-test.js

```diff
@@ -27,3 +27,17 @@ test('asyncStart waits for equal numbers of asyncEnd to finish a test', function
     adapter.asyncEnd();
   }, 50);
 });
+
+test('asyncStart should handle skipped tests that has no assert', function(assert) {
+  let FakeQUnitWithoutAssert = {
+    config: {
+      current: {}
+    }
+  };
+
+  const adapter = QUnitAdapter.create({ QUnit: FakeQUnitWithoutAssert });
```

Comment:
This needs to be updated to match (for the `this.qunit` vs `this.QUnit` change @scalvert 	suggested)

Reply:
Good call, updated.